### DISCUSSION TITLE
Linking Multiple Routes

### DISCUSSION THREAD

**Discussion by Chadlh** [osmandapp/OsmAnd#15003] [Maps Related]
I was wondering if there is any way to use multiple saved routes when setting up navigation from my location.  I've created multiple routes for motorcycle adventure but would like to choose which ones to use in a days adventure when starting navigation

**Comment by scaidermern** [comment]
You can use the plan a route tool to append multiple routes (tracks) to each other. This way you will create a new route based on multiple existing routes.

**Reply by Chadlh** [reply]
Thanks for suggesting that.  It worked well until it chose how to navigate.  When it chose where to navigate it avoided some of the route.
![Screenshot_20220803-164240_OsmAnd](https://user-images.githubusercontent.com/110543005/182695975-13cc6d23-dac5-4876-a660-69d60cc2cee5.jpg)

**Comment by scaidermern** [comment]
Looks like a bug. Create a new issue and attach the GPX file.

**Reply by vshcherb** [reply]
I don't think they are linked cause there are gaps between tracks. So ton screenshot they are still separate tracks. It's probably easier to draw new track over
